onnxruntime/python/tools/transformers/convert_generation.py

```diff
@@ -2211,7 +2211,7 @@ def test_gpt_model(args: argparse.Namespace, sentences: Optional[List[str]] = No
             dir = os.path.join(test_data_dir, "test_data_set_" + str(i))
             output_test_data(dir, inputs)
 
-    logger.debug("ORT inputs", inputs) # noqa: PLE1205
+    logger.debug("ORT inputs", inputs)
```

Comment:
```suggestion
    logger.debug("ORT inputs: %s", inputs)
```

Reply:
For it to be valid. Also avoid using f string in logs to avoid unnecessary string generation

Reply:
https://pylint.readthedocs.io/en/latest/user_guide/messages/error/logging-too-many-args.html